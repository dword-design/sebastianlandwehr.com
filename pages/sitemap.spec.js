import { getNuxtConfig } from '@dword-design/base-config-nuxt'
import { property } from '@dword-design/functions'
import tester from '@dword-design/tester'
import testerPluginNuxt from '@dword-design/tester-plugin-nuxt'
import axios from 'axios'
import pretty from 'pretty'

export default tester(
  {
    async init() {
      expect(
        axios.get('http://localhost:3000/sitemap.xml')
          |> await
          |> property('data')
          |> pretty
      ).toMatchSnapshot(this)
    },
  },
  [testerPluginNuxt(getNuxtConfig())]
)
