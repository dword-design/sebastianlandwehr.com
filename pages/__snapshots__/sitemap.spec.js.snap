// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sitemap init 1`] = `
"<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
<urlset xmlns=\\"http://www.sitemaps.org/schemas/sitemap/0.9\\" xmlns:news=\\"http://www.google.com/schemas/sitemap-news/0.9\\" xmlns:xhtml=\\"http://www.w3.org/1999/xhtml\\" xmlns:mobile=\\"http://www.google.com/schemas/sitemap-mobile/1.0\\" xmlns:image=\\"http://www.google.com/schemas/sitemap-image/1.1\\" xmlns:video=\\"http://www.google.com/schemas/sitemap-video/1.1\\">
  <url>
    <loc>http://localhost:3000/blog/creating-an-rss-feed-from-nuxt-content-with-full-body-html-code</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/generating-beautiful-flowchart-diagrams-with-mermaid-and-vue-nuxt</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/how-to-access-nuxt-js-page-data-in-route-meta-fields</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/how-to-write-file-based-javascript-tests-with-real-files</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/image-snapshot-testing-with-mocha</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/nuxt-content-how-to-keep-createdat-and-updatedat-valid-after-cloning</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/sending-emails-with-nuxt-js-the-easy-way</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/using-acss-atomic-css-framework-aka-atomizer-with-nuxt</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog/using-modernizr-with-nuxt-js-to-detect-browser-features</loc>
  </url>
  <url>
    <loc>http://localhost:3000/blog</loc>
  </url>
  <url>
    <loc>http://localhost:3000/legal-notice</loc>
  </url>
  <url>
    <loc>http://localhost:3000/privacy-policy</loc>
  </url>
  <url>
    <loc>http://localhost:3000/support-me</loc>
  </url>
  <url>
    <loc>http://localhost:3000/</loc>
  </url>
</urlset>"
`;
